Ast/src/Parser.cpp

```diff
@@ -2004,6 +2008,12 @@ AstExpr* Parser::parsePrimaryExpr(bool asStatement)
         {
             expr = parseFunctionArgs(expr, false, Location());
         }
+        else if (lexer.current().type == Lexeme::InterpStringBegin)
+        {
+            report(lexer.current().location, "Interpolated strings cannot be used alone to call a function. Wrap this in parentheses.");
```

Comment:
Bike shedding for a good messaging...

```suggestion
            report(lexer.current().location, "Function calls do not support string interpolation expressions without parentheses; consider wrapping in parentheses.");
```

Reply:
Now allowed with #615 if ratified.